params/protocol_params.go

```diff
@@ -139,11 +139,16 @@ const (
 	// TODO(asa): Make these operations less expensive by charging only what is used.
 	// The problem is we don't know how much to refund until the refund is complete.
 	// If these values are changed, "setDefaults" will need updating.
-	MaxGasForDebitFromTransactions      uint64 = 38 * 1000
 	ExpectedGasForDebitFromTransactions uint64 = 23 * 1000
-	MaxGasForCreditToTransactions       uint64 = 32 * 1000
-	MaxGasToReadErc20Balance            uint64 = 15 * 1000
-	MaxGasToReadTobinTax                uint64 = 50 * 1000
+	MaxGasForDebitFromTransactions      uint64 = 46 * 1000
```

Comment:
Can you add a comment explaining that this is currently a workaround where we're overestimating the gas?

Reply:
Done.